src/compose/service.ts

```diff
@@ -667,13 +675,13 @@ export class Service {
 		deviceName: string;
 		containerIds: Dictionary<string>;
 	}): Dockerode.ContainerCreateOptions {
-		const { binds, volumes } = this.getBindsAndVolumes();
+		const { binds, mounts, volumes } = this.getBindsMountsAndVolumes();
 		const { exposedPorts, portBindings } = this.generateExposeAndPorts();
 
 		const tmpFs: Dictionary<''> = {};
-		_.each(this.config.tmpfs, (tmp) => {
+		for (const tmp of this.config.tmpfs) {
```

Comment:
This is a NIT, but maybe using reduce makes the definition shorter?

```
const tmpFs: Dictionary<''> = this.config.tmpfs.reduce((res, tmp) => ({...res, [tmp]: ''}), {});
```

Reply:
Good catch!